client/components/DataManagement/DataManagementRow/index.jsx

```diff
@@ -0,0 +1,1062 @@
+    padding: 0;
+    margin-right: 8px;
+    border-radius: 50%;
+
+    &.issues {
+        background: #f2ba00;
+    }
+
+    &.reports-not-started {
+        background: #7c7c7c;
+    }
+
+    &.reports-in-progress {
+        background: #3876e8;
+    }
+
+    &.reports-complete {
+        background: #2ba51c;
+    }
+
+    &.reports-missing {
+        background: #ce1b4c;
+    }
+`;
+
+const NoneText = styled.span`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+
+    font-style: italic;
+    color: #6a7989;
+`;
+
+const DataManagementRow = ({
+    isAdmin,
+    ats,
+    testPlan,
+    testPlanVersions,
+    setTestPlanVersions
+}) => {
+    const { triggerLoad, loadingMessage } = useTriggerLoad();
+    const {
+        themedModal,
+        showThemedModal,
+        setShowThemedModal,
+        setThemedModalTitle,
+        setThemedModalContent,
+        setFocusRef
+    } = useThemedModal({
+        type: THEMES.WARNING,
+        title: 'Error Updating Test Plan Status'
+    });
+    const [updateTestPlanVersionPhaseMutation] = useMutation(
+        UPDATE_TEST_PLAN_VERSION_PHASE
+    );
+
+    // State
+    const [activePhases, setActivePhases] = useState({});
+    const [rdTestPlanVersions, setRdTestPlanVersions] = useState([]);
+    const [draftTestPlanVersions, setDraftTestPlanVersions] = useState([]);
+    const [candidateTestPlanVersions, setCandidateTestPlanVersions] = useState(
+        []
+    );
+    const [recommendedTestPlanVersions, setRecommendedTestPlanVersions] =
+        useState([]);
+
+    const [showAdvanceModal, setShowAdvanceModal] = useState(false);
+    const [advanceModalData, setAdvanceModalData] = useState({});
+
+    const draftVersionStringRef = useRef();
+    const candidateVersionStringRef = useRef();
+    const recommendedVersionStringRef = useRef();
+
+    useEffect(() => {
+        // TestPlanVersions separated by current TestPlan's phase
+        setActivePhases({});
+        setRdTestPlanVersions(
+            testPlanVersions.filter(({ phase }) => phase === 'RD')
+        );
+        setDraftTestPlanVersions(
+            testPlanVersions.filter(({ phase }) => phase === 'DRAFT')
+        );
+        setCandidateTestPlanVersions(
+            testPlanVersions.filter(({ phase }) => phase === 'CANDIDATE')
+        );
+        setRecommendedTestPlanVersions(
+            testPlanVersions.filter(({ phase }) => phase === 'RECOMMENDED')
+        );
+    }, [testPlanVersions]);
+
+    // Get the version information based on the latest or earliest date info from a group of
+    // TestPlanVersions
+    const getVersionData = (testPlanVersions, dateKey = 'updatedAt') => {
+        const earliestVersion = testPlanVersions.reduce((a, b) =>
+            new Date(a[dateKey]) < new Date(b[dateKey]) ? a : b
+        );
+        const earliestVersionDate = new Date(earliestVersion[dateKey]);
```

Comment:
I was playing around with the data management, and I was trying to figure out what would happen when we introduce a new draft version when there is already a recommended version. What happened was, the draft version never appeared in the data management page. Is that intended?

FYI the test plan I was looking at was the Disclosure Navigation Menu Example.

Reply:
You were right that there was an issue here! There was a missing check to verify that there is a newer version for the test plan to be shown in the row, if at a lower phase. https://github.com/w3c/aria-at-app/pull/713/commits/a4b2585475974f20fb10d955e33169103761a7dc should address this.